core/src/main/java/org/apache/iceberg/BaseFileScanTask.java

```diff
@@ -45,31 +47,66 @@ protected FileScanTask self() {
 
   @Override
   protected FileScanTask newSplitTask(FileScanTask parentTask, long offset, long length) {
-    return new SplitScanTask(offset, length, parentTask);
+    return new SplitScanTask(offset, length, parentTask, deletesSizeBytes());
```

Comment:
Why do we use the method `deleteSizeBytes` here? I can't get the difference from the 178th line.

Reply:
I gave a bit of explanation [here](https://github.com/apache/iceberg/pull/8346#discussion_r1301800846), let me know if that makes sense.

Reply:
Make sense. I got it.